ODP-Runtime.md

```diff
@@ -124,61 +121,54 @@ Compliance
 
 In order to locate common resources, some commonly set configuration details are necessary. 
 
--   All environment variables discussed in this section MUST be set on all nodes.
-> Comment (SCG): We need to discuss the scope of these variables. Are they automatically available to every user that logs into the machine, or is there a standard location in which an "-env.sh" script can be found to set the environment variables?
-
--   An ODP Platform MUST set the JAVA\_HOME.
-> Comment (SCG): We had discussed this being available via the "hadoop" script, is this requiring it to be set globally? 
+-   An ODP Platform MUST either explicitly set `JAVA_HOME` or configure it in `hadoop-env.sh` and `yarn-env.sh`. In a future specification, `yarn-env.sh` will be removed.
 
 -   ODP Platforms MUST have all of the base Apache Hadoop components installed.
 
--   ODP Platforms MUST pass the Big Top smoke tests.
-> Comment (AFG) I assume this means the ODP specific Big Top tests rather than generic Apache Big Top ones.  This should be made explicit.
+-   ODP Platforms MUST pass the Apache Big Top 1.0.0 Hadoop smoke tests.
 
 -   ODP Platforms MUST NOT change public APIs, where an API is defined as either a Java API (aka "Apache Hadoop ABI") or a REST API. See the [Apache Hadoop Compatibility guidelines](http://hadoop.apache.org/docs/r2.7.1/hadoop-project-dist/hadoop-common/Compatibility.html#Java_Binary_compatibility_for_end-user_applications_i.e._Apache_Hadoop_ABI) for more information.
 
--   ODP Platforms MUST NOT modify version strings output by Hadoop components, such as those displayed in log files, or returnd via public API's.
+-   ODP Platforms MUST modify the version string output by Hadoop components, such as those displayed in log files, or returned via public API's such that they contain `-(vendor string)` where `(vendor string)` is an appropriate entry for the ODP Platform vendor in the output.
 
--   An ODP Platform MUST keep the same basic directory layout and content as the equivalent Apache component. Changes to that directory layout MUST be enabled by the component itself with the appropriate configurations for that layout configured.
-> Comment (SCG): A clarifying example here would be useful
+-   An ODP Platform MUST keep the same basic directory layout and content as the equivalent Apache component. Changes to that directory layout MUST be enabled by the component itself with the appropriate configurations for that layout configured.  For example, if Apache Hadoop YARN's package distribution contains a share directory with content, then that share directory with the equivalent content must be preset.
```

Comment:
I'm torn about raising this, because it can be a thorny topic but....

What about contents of files?  Certain files (e.g. configuration files) are expected to have different contents than the ODP reference build or as produced from the original component build process, other files we expect to exactly match -- such as jar files.   We were playing with wording at some point about exactly matching the ODP reference implementation except where explicitly permitted by the spec.  Do we want to move forward with something like this?